site/docs/install.md

```diff
@@ -25,7 +28,7 @@ See the instructions for installing Bazel on:
 *   [Scoop](https://github.com/scoopinstaller/scoop-main/blob/master/bucket/bazel.json)
 
 ## Community-supported architectures
-*   [ppc64le](https://oplab9.parqtec.unicamp.br/pub/ppc64el/bazel)
 
+*   [ppc64le](https://oplab9.parqtec.unicamp.br/pub/ppc64el/bazel)
```

Comment:
I don't know what is correct here, but one use is ppc64**le** and the other is ppc64**el**. That seems off.

Reply:
Fixed to `el`.